src/sentry/profiles/profile_chunks.py

```diff
@@ -0,0 +1,41 @@
+from snuba_sdk import Column, Condition, Direction, Op, OrderBy, Query, Request, Storage
+
+from sentry import options
+from sentry.search.events.types import ParamsType
+from sentry.snuba.dataset import Dataset, StorageKey
+from sentry.snuba.referrer import Referrer
+from sentry.utils.snuba import raw_snql_query
+
+
+def get_chunk_ids(
+    params: ParamsType,
+    profiler_id: str,
+    project_id: int,
+) -> list[dict[str, str]]:
+    max_chunks = options.get("profiling.continuous-profiling.chunks-set.size")
+    query = Query(
+        match=Storage(StorageKey.ProfileChunks.value),
+        select=[
+            Column("chunk_id"),
+        ],
+        where=[
+            Condition(Column("start_timestamp"), Op.GTE, params.get("start")),
+            Condition(Column("end_timestamp"), Op.LT, params.get("end")),
```

Comment:
This might cause some alignment issues when rendering. For example, suppose we have profile chunks for the following time ranges
- 0 to 100
- 100 to 200

And the request asks for 50 to 150. Under in this situation, this query would return no results because
- for 0 to 100, the start timestamp 0 is less than the param start
- for 100 to 200, the end timestamp 200 is greater than the param end

I think the condition you really want here is

```suggestion
            Condition(Column("end_timestamp"), Op.GTE, params.get("start")),
            Condition(Column("start_timestamp"), Op.LT, params.get("end")),
```

Reply:
You're correct.

Good catch! thanks!